inspirehep/migrator/cli.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Manage migrator from INSPIRE legacy instance."""
+
+
+import sys
+from textwrap import dedent
+
+import click
+from flask import current_app
+from flask.cli import with_appcontext
+
+from .tasks import (
+    migrate_from_mirror,
+    migrate_record_from_legacy,
+    populate_mirror_from_file,
+)
+
+
+def halt_if_debug_mode(force):
+    message = """\
+    The application is running in debug mode, which leaks memory when doing
+    many database operations. To avoid problems, disable debug mode. This can
+    be done by setting "DEBUG=False" in the config or setting the environment
+    variable "APP_DEBUG=False". If you know what you are doing, you can pass
```

Comment:
is the `APP_` prefix still in use for env variables or is it `INVENIO_` now?

Reply:
both of them are working